src/styles/form.scss

```
@@ -0,0 +1,58 @@
+@use 'base' as base;
+
+.block-shadow{
+    box-shadow: 0px 0px 6px rgba(0, 0, 0, 0.14);
```

HEX тоже позволяет задействовать альфа-канал. Можно использовать его для единообразия
```suggestion
    box-shadow: 0px 0px 6px #00000024;
```
https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4